File: Exercism_Javascript_Track/rna-transcription/rna-transcription.js
//
// This is only a SKELETON file for the 'RNA Transcription' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const toRna = (nucleotide) => {
    let Rna = '';

    for(let i =0; i<nucleotide.length; i++){

        if(nucleotide.charAt(i) === 'G'){
            Rna +='C';
        }
        else if(nucleotide.charAt(i)=== 'C'){
            Rna +='G';
        }
        else if(nucleotide.charAt(i)=== 'T'){
            Rna +='A';        
        }
        else if(nucleotide.charAt(i)=== 'A'){
            Rna +='U';        
        }
      }
    
      return Rna;

};

File: Exercism_Go_Track/rna-transcription/rna_transcription.go
package strand

func ToRNA(dna string) string {
    rna := ""
	for _, char := range dna{
        switch char {
            case 'G':
        		rna += string('C')
            case 'C':
        		rna += string('G')
            case 'T':
        		rna += string('A')
            case 'A':
        		rna += string('U')
        }
    }
	return rna
}
